static/js/modules/game/articleRenderer.js

```diff
@@ -86,12 +81,16 @@ export class ArticleRenderer {
             });
 
             // Remove "/wiki/" from string
-            this.loadPageWrapper(linkEl.getAttribute("href").substring(6))
+            this.loadPage(linkEl.getAttribute("href").substring(6))
         }
     }
 }
 
-
+function setMargin(frame) {
+    const element = document.getElementById("time-box");
+    let margin = (element.offsetHeight + 25) > 250 ? (element.offsetHeight + 25) : 250;
+    frame.style.marginBottom = margin +"px";
+}
```

Comment:
Depending on how we choose to format mobile, the timebox element's ID might not always be "time-box"

Reply:
Wouldn't the element ID stay the same? Or are you saying we might want to create/use a separate component for mobile?

Reply:
Yeah, I wanted to potentially keep creating a separate component for mobile as an option, depending on how different it ends up being compared to the desktop one. 

But I do think that might be part of a greater discussion, and we can just deal with margin resize when the time comes lol

Reply:
Ah kk, yea we can discuss that